src/Orleans/IDs/ActivationAddress.cs

```diff
@@ -32,6 +33,7 @@ internal class ActivationAddress
         public GrainId Grain { get; private set; }
         public ActivationId Activation { get; private set; }
         public SiloAddress Silo { get; private set; }
+        public MultiClusterStatus Status { get; private set; }
```

Comment:
I think you now also need to change the way `ActivationAddress` is serialized in Orleans serializer. We have manually written built in serializer for all runtime types. Look at BuiltInTypes.